src/scene_battle_rpg2k.cpp

```diff
@@ -801,14 +801,17 @@ void Scene_Battle_Rpg2k::CreateExecutionOrder() {
 }
 
 void Scene_Battle_Rpg2k::CreateEnemyActions() {
-	std::vector<Game_Battler*> active_enemies;
-	Main_Data::game_enemyparty->GetActiveBattlers(active_enemies);
+	std::vector<Game_Battler*> enemies;
+	Main_Data::game_enemyparty->GetBattlers(enemies);
+
+	for (Game_Battler* battler : enemies) {
+		if (!battler->CanAct()) {
```

Comment:
I forgot something here: When the enemy can't Act the action must be a "NoMove" because otherwise auto-healing of states doesn't work.

Correct code should be:
```cpp
battler->SetBattleAlgorithm(std::make_shared<Game_BattleAlgorithm::NoMove>(battler));
ActionSelectedCallback(battler);
continue;
```

Maybe @carstene1ns can amend this to his "little cosmetics" PR because thats the only one still open :D